src/App.js

```diff
@@ -222,6 +226,8 @@ function App() {
         isOpen={modalIsOpen}
         onClose={setModalIsOpenToFalse}
         onKeyDown={HandleKeyDown}
+        playTime={playTime}
```

Comment:
So we are still passing the function to VidModal. playTime may not be the best name to represent its value but it represents the currentTime for the video in VidModal to be set to when rendered.
```
const [state, setState] = useState()
...

const handleModalOpen = () => {
    //set isopen true
    // calculate & set modal time
}
...
playTime={state}// this is the calculated time in seconds like: 23.1234
...
```

Take a look how VidModal handles playTime

Reply:
Would getPlayTime be a  better name?